doc/manuals/install_guide.rst

```diff
@@ -77,6 +77,13 @@ the last couple of steps, Fletch should build without any issues.
 
 `make`
 
+.. note::
```

Comment:
There are some RST syntax issues here.  At least with the way Github renders this file.  The work "note" is rendered on a blank like and looks just like the command "make" above it.  Its not clear where the command line examples end and that "note" is not a command you should type.  To be fair, this might be more of an issue with the syntax used in the commands above.

Reply:
Oh, geez, *seriously* github? 😒